### PR TITLE
Support php 7.2

### PR DESCRIPTION
**Pull request by asbiin**
You don't really need php 7.4.
As [Laravel minimum php version is 7.2.5](https://laravel.com/docs/7.x/installation#server-requirements), I propose you to go back to that version.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
You'd need to update the CI config too.

**Comment by asbiin** [comment]
@GrahamCampbell I've activated tests for php 7.2 and 7.3. Phpunit 9.0 requires php 7.3, so I've downgraded it too.

**Comment by GrahamCampbell** [comment]
Better to allow both versions. Also, :-1: on the other changes you made to the config. Fast finish is a bad idea because it doesn't let you see the full picture if something fails. It was only useful on Travis as a measure to compat their job limits and waiting times.

**Comment by asbiin** [comment]
OK no problem, I've reverted the fail-fast option change. It will [default to true](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast), which is fine!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,7 +8,6 @@ jobs:
 
     strategy:
       max-parallel: 5
```
Comment (GrahamCampbell) [inline .github/workflows/ci.yml]: This should be removed too.

File: composer.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
         "graham-campbell/analyzer": "^3.0",
         "graham-campbell/testbench": "^5.4",
         "mockery/mockery": "^1.3",
-        "phpunit/phpunit": "^9.0",
+        "phpunit/phpunit": "^8.5",
```
Comment (GrahamCampbell) [inline composer.json]: Should be `^8.5|^9.0`.

**Comment by GrahamCampbell** [comment]
I still, actually, doubt this PR will be accepted, anyway. Maybe there is a possibility it will be accepted if you just lowered to `^7.3`, though. I will be forking this repo to restore 7.3 support anyway, if it isn't added here.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (asbiin) [reply .github/workflows/ci.yml]: This allows you to run each testsuite in parallel, and reduce test time!
But I'll remove it ok.

File: composer.json [hunk 3]
```diff
@@ -23,7 +23,7 @@
         "graham-campbell/analyzer": "^3.0",
         "graham-campbell/testbench": "^5.4",
         "mockery/mockery": "^1.3",
-        "phpunit/phpunit": "^8.5",
+        "phpunit/phpunit": "^8.5|^9.0",
```
Comment (vinkla) [inline composer.json]:
```suggestion
        "phpunit/phpunit": "^8.5 || ^9.0",
```

**Comment by vinkla** [comment]
> You don't really need php 7.4.
> As Laravel minimum php version is 7.2.5, I propose you to go back to that version.

The question is, are you using this library with PHP 7.3 or lower?

**Comment by GrahamCampbell** [comment]
I am using it with PHP 7.3 and 7.4. Not 7.2 anymore, since December 2019.

**Comment by asbiin** [comment]
@vinkla I'm using this package on an application, where I don't want to require php >= 7.3, while Laravel support php >= 7.2.5.
If I require php 7.3, there is a big probability that some users will [complain about it](https://github.com/monicahq/monica/issues/3037). I would like to continue using laravel-hashids library ...

**Comment by vinkla** [comment]
I'm generally against supporting older versions of Laravel and PHP. We as a community should strive for keeping our projects up to date with the latest software updates. With that said, in this case this package has about 70k+ downloads each month. Maybe we could make an exception.

**Pull request merged**